### DISCUSSION TITLE
AWS Hasura becomes unresponsive, then ok again for a while!

### DISCUSSION THREAD

**Discussion by paulfentonfuse** [hasura/graphql-engine#9181] [Question]
I have launched an instance of container definition hasura/graphql-engine:v2.14.0 to AWS as a FARGATE ECS service with one task and resources CPU 256 and memory 1024. It is connected to a RDS postgres database within AWS.

I can successfully connect to the Hasura console and my schema is perfect for my database.

However, every few minutes, both the console and external graphql interfacing becomes unresponsive. In the console, the red progress bar at the top hovers at around 95% and then suddenly completes after about 60-90 secs.

Hasura will then operate normally for a few minutes and then the whole thing repeats!

I have other Hasura instances of the same spec that work perfectly fine, but this one is problematic.

I have searched the RDS instance logs, ECS container logs and nothing appears to be generating any errors.

Has anyone else experienced this and what else can I look for?? Appreciated in advance!

**Comment by meetzaveri** [comment]
Hey @paulfentonfuse 

Can we know which edition of hasura you are using ? Is it open source (community edition) or enterprise edition (EE) ?

**Reply by paulfentonfuse** [reply]
community

**Reply by meetzaveri** [reply]
Okay, let us perform some replication and test this scenario in AWS ECS.

**Reply by paulfentonfuse** [reply]
Thanks @meetzaveri 
Here is my service creation script and attached is my task def (with credentials redacted)

aws ecs create-service \
--cluster fs3mt \
--service-name hasura_mt \
--task-definition hasura_mt:11 \
--desired-count 1 \
--launch-type "FARGATE" \
--health-check-grace-period-seconds 60 \
--service-registries "registryArn=arn:aws:servicediscovery:ap-southeast-2:XXXXXXXXXXXX:service/srv-ylpodzaznzlfmuuu,containerName=hasura_mt" \
--load-balancers "targetGroupArn=arn:aws:elasticloadbalancing:ap-southeast-2:XXXXXXXXXXXX:targetgroup/hasuramt/1c19443433286bd976,containerName=hasura_mt,containerPort=8080" \
--network-configuration \
"awsvpcConfiguration={subnets=[subnet-XXXXXXXXXXXX,subnet-XXXXXXXXXXXX,subnet-XXXXXXXXXXXX],securityGroups=[sg-XXXXXXXXXXXX]"} \
--profile XXXXXXXXXXXX_dev --region ap-southeast-2

[hasura-task.txt](https://github.com/hasura/graphql-engine/files/9935027/hasura-task.txt)

**Reply by meetzaveri** [reply]
hey @paulfentonfuse , can you aid something here through which can know the cause ?

We want to know the actual behavior, like if you can try to confirm if the UI / console freezes for some time or is it the HTTP(s) requests taking more time ?

**Reply by paulfentonfuse** [reply]
@meetzaveri The console and requests all freeze for some time. The red progress bar at the top of the console gets stuck on about 95% and freezes there for about 60 secs. It then completes the request and shows your data.
This not only happens when accessing the console to retrieve data, but also when just accessing the /console home page.
